File: corpus/typescript/fixture_07.ts
type CartItem7 = { price: number; qty?: number };
export function calculateDiscount7(cartItems: CartItem7[], taxRate: number): number {
  const totalAmount = cartItems.reduce((sum, item) => sum + item.price * (item.qty ?? 1), 0);
  return totalAmount > 70 ? totalAmount * (1 + taxRate) : totalAmount;
}

export class PriceBook7 {
  lookupPrice(sku: string): number { return sku.length + 7; }
}

File: corpus/python/fixture_07.py
def calculate_discount_7(cart_items, tax_rate):
    total_amount = sum(item["price"] * item.get("qty", 1) for item in cart_items)
    if total_amount > 70:
        return total_amount * (1 + tax_rate)
    return total_amount

class PriceBook7:
    def lookup_price(self, sku):
        return len(sku) + 7
